scripts/publisherDVL1000.py

```diff
@@ -6,295 +6,354 @@
+        BottomBeamFom3Valid = theData["frames"][5]["inputs"][1]["lines"][2]["valid"]
+        BottomBeamFom4Valid = theData["frames"][5]["inputs"][1]["lines"][3]["valid"]
+
+        # Beam Dist Variables
+        BottomBeamDistMin = theData["frames"][5]["inputs"][2]["min"]
+        BottomBeamDistMax = theData["frames"][5]["inputs"][2]["max"]
+        BottomBeamDistUnit = theData["frames"][5]["inputs"][2]["units"]
+        BottomBeamDist1Data = theData["frames"][5]["inputs"][2]["lines"][0]["data"][0]
+        BottomBeamDist2Data = theData["frames"][5]["inputs"][2]["lines"][1]["data"][0]
+        BottomBeamDist3Data = theData["frames"][5]["inputs"][2]["lines"][2]["data"][0]
+        BottomBeamDist4Data = theData["frames"][5]["inputs"][2]["lines"][3]["data"][0]
+        BottomBeamDist1Valid = theData["frames"][5]["inputs"][2]["lines"][0]["valid"]
+        BottomBeamDist2Valid = theData["frames"][5]["inputs"][2]["lines"][1]["valid"]
+        BottomBeamDist3Valid = theData["frames"][5]["inputs"][2]["lines"][2]["valid"]
+        BottomBeamDist4Valid = theData["frames"][5]["inputs"][2]["lines"][3]["valid"]
+
+        # XYZ Velocity Variables
+        BottomXyzVelMin = theData["frames"][6]["inputs"][0]["min"]
+        BottomXyzVelMax = theData["frames"][6]["inputs"][0]["max"]
+        BottomXyzVelUnit = theData["frames"][6]["inputs"][0]["units"]
+        BottomXyzVel1Data = theData["frames"][6]["inputs"][0]["lines"][0]["data"][0]
+        BottomXyzVel2Data = theData["frames"][6]["inputs"][0]["lines"][1]["data"][0]
+        BottomXyzVel3Data = theData["frames"][6]["inputs"][0]["lines"][2]["data"][0]
+        BottomXyzVel4Data = theData["frames"][6]["inputs"][0]["lines"][3]["data"][0]
+        BottomXyzVel1Valid = theData["frames"][6]["inputs"][0]["lines"][0]["valid"]
+        BottomXyzVel2Valid = theData["frames"][6]["inputs"][0]["lines"][1]["valid"]
+        BottomXyzVel3Valid = theData["frames"][6]["inputs"][0]["lines"][2]["valid"]
+        BottomXyzVel4Valid = theData["frames"][6]["inputs"][0]["lines"][3]["valid"]
+
+        # XYZ FOM Variables
+        BottomXyzFomMin = theData["frames"][6]["inputs"][1]["min"]
+        BottomXyzFomMax = theData["frames"][6]["inputs"][1]["max"]
+        BottomXyzFomUnit = theData["frames"][6]["inputs"][1]["units"]
+        BottomXyzFom1Data = theData["frames"][6]["inputs"][1]["lines"][0]["data"][0]
+        BottomXyzFom2Data = theData["frames"][6]["inputs"][1]["lines"][1]["data"][0]
+        BottomXyzFom3Data = theData["frames"][6]["inputs"][1]["lines"][2]["data"][0]
+        BottomXyzFom4Data = theData["frames"][6]["inputs"][1]["lines"][3]["data"][0]
+        BottomXyzFom1Valid = theData["frames"][6]["inputs"][1]["lines"][0]["valid"]
+        BottomXyzFom2Valid = theData["frames"][6]["inputs"][1]["lines"][1]["valid"]
+        BottomXyzFom3Valid = theData["frames"][6]["inputs"][1]["lines"][2]["valid"]
+        BottomXyzFom4Valid = theData["frames"][6]["inputs"][1]["lines"][3]["valid"]
```

Comment:
Ugly wall of code with a repeating pattern (min, max, unit, data, valid). Could it maybe be shortened by placing variable ptrs in a list and looping thru them instead?